scripts/docker-run.sh

```diff
@@ -38,6 +38,29 @@ else
 	DOCKER_RUN="docker run"
 fi
 
+#
+# Set UPSTREAM_BRANCH. This will determine which version of the linux package
+# mirror is used.
+#
+if [[ -z "$UPSTREAM_PRODUCT_BRANCH" ]]; then
+    echo "UPSTREAM_PRODUCT_BRANCH is not set."
+    if ! source "$TOP/branch.config" 2>/dev/null; then
+        echo "No branch.config file found in repo root."
+        exit 1
+    fi
+
+    if [[ -z "$UPSTREAM_BRANCH" ]]; then
+        echo "UPSTREAM_BRANCH parameter was not sourced from branch.config." \
+             "Ensure branch.config is properly formatted with e.g." \
+             "UPSTREAM_BRANCH=\"<upstream-product-branch>\""
+        exit 1
+    fi
+    echo "Defaulting to branch $UPSTREAM_BRANCH."
```

Comment:
nit: "Defaulting to branch $UPSTREAM_BRANCH set in branch.config".

Reply:
updated